packages/eslint-config-react/readme.md

```diff
@@ -1,37 +0,0 @@
-# @strv/eslint-config-react
```

Comment:
Let's bring back at least some basic version of the readme - it will be visible on npmjs.com once we publish this, leaving it empty looks unprofessional. 💼

Reply:
Added. 🎨